### ISSUE TITLE
[Win] Hyper-V validation should not be done under Home SKUs

### ISSUE THREAD

**Issue by jeromelaban** [unoplatform/uno.check#115]
The current Hyper-V check fails when run under Windows 10/11 Home editions (and some others) and should be disabled in this case.

**Comment by DevTKSS** [comment]
@jeromelaban do we need the hyper-V check to success when targeting desktop? At work I would not use hyper-V and android or any wsl thing, but would like to use SkiaRenderer + Desktop target instead of the windows native target. Do you see any options to make this check be exclude-able without constantly typing in the skip names?

**Comment by jeromelaban** [comment]
No, the check is not needed when selecting desktop only, it's only needed for Android.

**Comment by DevTKSS** [comment]
@jeromelaban hm, so then it would be interesting why this is popping up using the `--tfm net9.0-desktop` flag just like the IDE is doing 🤔 On my current device I have it set up, so you can see this in this beeing marked as success, but as mentioned, I would rather not want it to be checked and fail just because I am targeting desktop if this could be made possible?

[Uno Platform Check-tfm-desktop.txt](https://github.com/user-attachments/files/21353635/Uno.Platform.Check-tfm-desktop.txt)

**Comment by jeromelaban** [comment]
Indeed, the check is not needed in desktop and we should make a change that disables it for that target.
